backstory/src/main/java/com/google/sps/perspective/PerspectiveValue.java

```diff
@@ -0,0 +1,59 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data.perspective;
+
+import au.com.origma.perspectiveapi.v1alpha1.models.AttributeType;
+import java.util.Map;
+
+/**
+ * A container to hold text and the scores generated by that text.
+ */
+public class PerspectiveValue {
```

Comment:
A more descriptive name for this class is probably the move. PerspectiveValue is a little vague considering this is more specifically a container of many perspective values for pre-tagged text. Perhaps PerspectiveValueContainer or PerspectiveValues would be more applicable.

Reply:
PerspectiveValues sounds good to me. I'll make the change.

Reply:
Done